libcxx/test/libcxx/transitive_includes/cxx20.csv

```diff
@@ -682,6 +682,7 @@ ranges optional
 ranges span
 ranges tuple
 ranges type_traits
+ranges variant
```

Comment:
Is this necessary? Why does that happen?

Reply:
`variant` is used as an implementation detail of `common_iterator`, which is used by `common_view`. It probably didn't show up before because it's been swallowed by `iterator`.